worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerReporter.java

```diff
@@ -111,22 +111,31 @@ public void reportJobTaskProgress(final String jobTaskId, final int estimatedPer
     /**
      * Reports the specified job task as complete.
      *
-     * @param jobTaskId identifies the completed job task
+     * @param jobTaskIds identifies the completed job tasks
      * @return JobTrackingWorkerDependency list containing any dependent jobs that are now available for processing
      * @throws JobReportingException
      */
     @Override
-    public List<JobTrackingWorkerDependency> reportJobTaskComplete(final String jobTaskId) throws JobReportingException
+    public List<JobTrackingWorkerDependency> reportJobTaskComplete(final List<String> jobTaskIds) throws JobReportingException
     {
-        LOG.debug("Reporting completion of job task {}...", jobTaskId);
+        LOG.debug("Reporting completion of job task {}...", jobTaskIds.get(0));
```

Comment:
This is only logging the first id.

Check if this works:
```suggestion
        LOG.debug("Reporting completion of job task {}...", jobTaskIds);
```